test/test_unit_client.py

```diff
@@ -128,15 +128,17 @@ def test_client_passes_default_account_to_api_initializer_if_none_is_provided(se
             url='http://myurl',
         )
 
+    @patch('conjur.wrapper.keystore_adapter.KeystoreAdapter.is_keyring_accessible', return_value=False)
     @patch('conjur.api.client.Api')
-    def test_client_performs_password_api_login_if_password_is_provided(self, mock_api_instance):
+    def test_client_performs_password_api_login_if_password_is_provided(self, mock_api_instance, mock_accessible):
```

Comment:
Can you elaborate the use of mock_accessible ?

Reply:
mock_accessible means that the keyring isn't accessible and because it is a unit test it doesnt matter which flow we use (keyring or netrc). I use this so I can mock the load function that is called from the Client because for this test we are just testing that the API gets the parameters passed from the Client